source/process/sg_grammar-spelling-mechanics.rst

```diff
@@ -0,0 +1,148 @@
+Grammar, spelling, and mechanics
+================================
+
+To maintain consistency across all Mattermost technical documentation, adhere to the guidelines here.
+
+Language and spelling
+---------------------
+
+Write documents in English. Use American spelling.
+
+Paragraphs and sentences
+------------------------
+
+Paragraphs should express one idea or topic. Long paragraphs are sometimes difficult to read on screen, so try to keep them to 5 sentences or less. Short paragraphs are easier for people to scan quickly.
+
+Try to keep sentences to 25 words or less in length. Short, single-clause sentences are often easier to understand and easier to translate.
```

Comment:
could referencing the [Hemingway writing app](http://www.hemingwayapp.com/) be useful?

It's a common tool to simplify your paragraphs and sentences, which could be helpful for anyone working on docs.

Not sure if you've used the app yourself or if you've used something else before @JeffSchering